stream.js

```diff
@@ -1,3 +1,17 @@
+;(function(root, factory, undefined){
+	
```

Comment:
was thinking of moving to a bundler but this can go in for now and we can improve as we go.